suite-native/atoms/src/Modal/BottomModal.tsx

```diff
@@ -56,32 +127,47 @@ export const BottomModal = ({
 }: BottomModalProps) => {
     const { applyStyle } = useNativeStyles();
     const insets = useSafeAreaInsets();
+    const { animatedModalWrapperStyle, closeModalAnimated, resetModalAnimated, panGestureEvent } =
+        useBottomModalAnimation({ onVisibilityChange });
+
+    useEffect(() => {
+        if (isVisible) {
+            resetModalAnimated();
+        }
+    }, [isVisible, resetModalAnimated]);
 
-    const handleCloseModal = () => onVisibilityChange(false);
+    const handleCloseModal = () => {
```

Comment:
Probably better to `useCallback` it since the component is rapidly updated.

Reply:
It is not, animation if done correctly doesn't rerender component.